OmsAgent/omsagent.py

```diff
@@ -943,6 +943,16 @@ def run_command_and_log(cmd, check_error = True, log_cmd = True):
         hutil_log_info('Output of command "{0}": \n{1}'.format(cmd, output))
     else:
         hutil_log_info('Output: \n{0}'.format(output))
+
+    # also write output to STDOUT since WA agent uploads that to Azlinux Kusto DB
+    try:
+        if sys.version_info >= (3,):
```

Comment:
use print(output) it works on both python 2.x and 3.x

Reply:
as per doc, print with parantheses is supported only in python 3

Reply:
print in python 2 doesn't take parentheses https://docs.python.org/2/tutorial/inputoutput.html

Reply:
print in python 2 doesn't take parentheses https://docs.python.org/2/tutorial/inputoutput.html